test/Microsoft.Azure.SignalR.AspNet.Tests/E2ETest/TestClientSet.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNet.SignalR.Client;
+using Microsoft.Azure.SignalR.Tests.Common;
+
+namespace Microsoft.Azure.SignalR.AspNet.Tests
+{
+    public class TestClientSet : ITestClientSet
+    {
+        private IList<HubConnection> _connections;
+        private IList<IHubProxy> _proxies;
+
+        public int Count {
+            get
+            {
+                if (_connections == null)
```

Comment:
>_connections [](start = 20, length = 12)

`_connections?.Count??0`?

Reply:
since I'd like to check if the connection is null or not.